app/players/edit.html

```diff
@@ -1,9 +1,14 @@
 <div class="edit-players">
-	<p>One character per line. Blank line to separate different parties. Format: name initiative hp</p>
+	<p>One character per line. Blank line to separate different parties. Format:<br/>
```

Comment:
Please move the first line to its own line (i.e. <p> should be on a line by itself) and indent the contents of the <p> tag.

Thanks!